src/validations/rules/ssp.sch

```diff
@@ -3450,9 +3450,66 @@
                 role="error"
                 test="@href">The import-profile element has a reference.</sch:assert>
         </sch:rule>
+        <sch:rule
+            context="oscal:control-implementation">
+            <sch:let
+                name="sensitivity-level"
+                value="/ => lv:sensitivity-level()" />
+            <sch:let
+                name="selected-profile"
+                value="$sensitivity-level => lv:profile()" />
+            <sch:let
+                name="required-controls"
+                value="$selected-profile/*//oscal:control/@id ! xs:string(.)" />
+            <sch:let
+                name="implemented-controls"
+                value="oscal:implemented-requirement/@control-id ! xs:string(.)" />
+            <sch:let
+                name="technical-controls"
+                value="$fedramp-values//fedramp:value-set[@name eq 'technical-control-id']//fedramp:enum/@value ! xs:string(.)" />
+            <sch:let
+                name="required-technical-controls"
+                value="$technical-controls[. = $required-controls]" />
+            <sch:let
+                name="missing-required-technical-controls"
+                value="$required-technical-controls[not(. = $implemented-controls)]" />
+            <sch:assert
+                diagnostics="technical-control-exists-diagnostic"
+                id="technical-control-exists"
+                role="error"
+                test="
+                    every $item in $missing-required-technical-controls
+                        satisfies exists(//oscal:implemented-requirement[@control-id eq $item])">Every required technical control is
```

Comment:
The bind variable is wrong. It should have been `$required-technical-controls`. As is, it is trying to find missing+required technical controls which are missing and thus will not be found.

The assertion test can be simplified to `count($missing-required-technical-controls) eq 0` or an equivalent.

Reply:
fixed

Reply:
I fixed this incorrectly into the last commit.  This is fixed (3rd commit) and working locally now.